lang/Cargo.toml

```diff
@@ -59,6 +59,5 @@ borsh = ">=0.9, <0.11"
 bytemuck = "1"
 solana-program = "1.16"
 thiserror = "1"
-
-# TODO: Remove. This crate has been added to fix a build error with the 1.16.0 release.
-getrandom = { version = "0.2", features = ["custom"] }
+# TODO: Remove once https://github.com/solana-labs/solana/issues/33504 is resolved.
+ahash = "0.8.4"
```

Comment:
This is no longer needed and was removed in https://github.com/coral-xyz/anchor/pull/2867